layouts/partials/page-meta-links.html

```diff
@@ -1,6 +1,6 @@
 {{/* template adapted from Docsy theme */}}
-{{ if .Path }}
-  {{ $pathFormatted := replace .Path "\\" "/" }}
+{{ if .File.Path }}
+  {{ $pathFormatted := replace .File.Path "\\" "/" }}
```

Comment:
I'd swap the commit order: fix this first.

@Rishit-dagli this change on its own might be OK to merge in a separate PR; what do you think?

Reply:
Thanks for pointing out that this commit is similar to https://github.com/kubernetes/website/pull/34360#issuecomment-1159031205 , I didn't notice this: my motivation to make this change was a failing build after https://github.com/kubernetes/website/pull/34361/commits/deacef302026e327cda4290ab14e2e3a70e61fdb

Now that #34360 is merged should I revert https://github.com/kubernetes/website/pull/34361/commits/0d732354e5ee53e9e6c5dcd0f0140ee0fad09f82 ?

Reply:
I believe deleting https://github.com/kubernetes/website/commit/0d732354e5ee53e9e6c5dcd0f0140ee0fad09f82 and force pushing might make more sense since #34360 will be merged before this